### DISCUSSION TITLE
Where is the list view of blocks for new editor?

### DISCUSSION THREAD

**Discussion by landwire** [woocommerce/woocommerce#39328] [WooCommerce New Product Editor] [answered]
I am missing the list view from the block editor when switching on the new product editor.
Any ideas how to get it back?

**Comment by landwire** [comment]
<img width="931" alt="Screenshot 2023-07-19 at 23 41 09" src="https://github.com/woocommerce/woocommerce/assets/1001202/d0dff7cb-2368-4b7f-95c3-0333426189c6">

**Comment by louwie17** [comment, marked as answer]
Hi @landwire,
We have actually added the List View as part of the latest WooCommerce release ( 8.0 ). Let us know if there are any other features you are missing in the description block editor.
